src/ngraph/op/fused/normalize_l2.cpp

```diff
@@ -71,9 +71,11 @@ NodeVector op::NormalizeL2::decompose_op() const
     const Shape input_shape{data.get_shape()};
 
     // Reshape to 4D tensor.
+    size_t axes_shift = 0;
```

Comment:
Please add a comment explaining why we need to use this `axes_shift` variable.

Reply:
Better yet, remove it. The op should be rank-independent.

Reply:
I close this PR. The entire decompose method contains bugs and constraints. I've created ticket on @mbencer